### PR TITLE
(Add pt-br language) Fixes pullrequest of DavidWilliamBalbino 

### PR DESCRIPTION
**Pull request by periscuelo**
I made this pullrequest just to help make the merge of a translation as fast as possible. Thank you all for looking it for us. If need more fixes please, let me know.

Thank you again

### CODE REVIEW

File: docs/pt-br/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Vuex
+
+> Nota: para os usu�rios TypeScript, vuex@3.0+ requer vue@2.5+ e vice-versa.
+
+- [Notas de vers�o] (https://github.com/vuejs/vuex/releases)
```
Comment (ktsn) [inline docs/pt-br/SUMMARY.md]: Can we remove the space between `]` and `(`? It's not rendered as link on generated docs (You can confirm it by `npm i -g gitbook-cli && npm run docs` on local)

Comment (periscuelo) [reply docs/pt-br/SUMMARY.md]: Thank you  so much for the Quick Tip. I go fix it right now.

Comment (periscuelo) [reply docs/pt-br/SUMMARY.md]: Fixed. If have something more please tell me. Thanks

**Comment by periscuelo** [comment]
Fixed files encode. I saw it after npm run docs tip. Thanks.

**Comment by periscuelo** [comment]
@ktsn I believe it's all reviewed now. But it may be that something has been forgetd. At least now you will not have to see a line tell me, I fix, one more line.....
There was more need for translation reviews than I thought. A lot of translation came straight from Google Translator....
If have any other thing did you look, and need change, tell me please.
Thanks.

**Comment by ktsn** [comment]
@periscuelo Thank you!

It might be too early to merge this PR but if anyone wants to improve the translation, please make a new PR.

**Pull request merged**